### PR TITLE
Container installation jcayouette

### PR DESCRIPTION
**Pull request by jcayouette**
# Container installation docs
 To support new container deployment features for both Uyuni and SUMA 5.0
 Target beta1 for basic use

## Task Overview

- [x] Uyuni will contain legacy installation documentation along side the new container deployment. 
- [x] SUMA 5.0 will include containerized deployment documentation while deprecating legacy installation documentation)

## Work in progress

- [x] Includes podman installation (For both suma and uyuni **ifdef required**)
- [x] Uyuni container utiliies installation (for both suma and uyuni **ifdef required**)
- [x] Persistent volumes 
- [x] Uyuni deployment as a container
- [x] SUMA deployment as a container
- [x]  SLE Micro 5.5 Deployment

# Target branches

- master (Uyuni)
- manager-5.0

# Links
https://github.com/uyuni-project/uyuni/tree/master/containers/doc
https://github.com/uyuni-project/uyuni-docs/pull/2190
https://github.com/SUSE/spacewalk/issues/22105
https://github.com/SUSE/spacewalk/issues/23125#issuecomment-1894327395

### CODE REVIEW

File: modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+:uyuni-content: true
+
+//The process of installing {productname} server was changed in version {productnumber}. 
+//{productname} is now deployed as a container.
+
+
+ifeval::[{uyuni-content} == true]
+== Deployment preparations
+
+In this section, you will gain expertise in setting up and deploying a {productname} Server. The process encompasses the installation of [command]``Podman``, [filename]``Uyuni container utilities``, deployment, and then initiating your interaction with the container through [command]``mgrctl``.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc:13]:
```suggestion
In this section, you will gain expertise in setting up and deploying a {productname} Server.
The process encompasses the installation of [command]``Podman``, [systemitem]``Uyuni container utilities``, deployment, and then initiating interaction with the container through [command]``mgrctl``.
```

File: modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+It is important to note that this section assumes you have already configured a {tumbleweed} or {opensuse} {opensuse-version} server, whether it's running on a physical machine or within a virtual environment.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc:15]:
```suggestion
[NOTE]
====
This section assumes you have already configured a {tumbleweed} or {opensuse} {opensuse-version} server, whether it is running on a physical machine or within a virtual environment.
====
```

File: modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+. Install [command]``Podman`` by running the following command:
++
+
+[source,shell]
+----
+sudo zypper -y install podman
+----
+
+. Zypper will download and install [command]``Podman`` along with its dependencies.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc:43]:
```suggestion
+

Zypper will download and install [command]``Podman`` along with its dependencies.
```
This is not a separate step, but result of the previous one ;)

File: modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+. Once the installation is complete, verify that [command]``Podman`` is installed by running:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc:45]:
```suggestion
. When the installation is complete, verify that [command]``Podman`` is installed by running:
```

File: modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+[source,shell]
+----
+podman --version
+----
++
+
+This command should display the installed version of [command]``Podman``.
+You should now have [command]``Podman`` successfully installed on your openSUSE Leap 15.5 system, and you can start using it to manage containers on your system.
+
+For more information see, link:https://podman.io/[https://podman.io]
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc:57]:
```suggestion
For more information, see https://podman.io/.
```

**Review by keichwa — CHANGES_REQUESTED** [review]
I'll continue later.

**Comment by jcayouette** [comment]
This still needs a lot of work but since it is merging to master for Beta 1 I will merge for now and continue working on it. We can review the next PR.

**Pull request merged**